lib/led/rgb.js

```diff
@@ -405,4 +419,18 @@ RGB.prototype.stop = function() {
   return this;
 };
 
+RGB.prototype.intensity = function(intensity) {
+  var state = priv.get(this);
+
+  if (arguments.length === 0) {
+    return state.intensity;
+  }
+
+  state.intensity = Math.min(100, Math.max(0, intensity));
```

Comment:
This pattern is also defined in fn.js. https://github.com/rwaldron/johnny-five/blob/master/lib/fn.js#L48-L50 Maybe we should call that for consistency and readability?